### DISCUSSION TITLE
mode standalone js file too large

### DISCUSSION THREAD

**Discussion by niuzhenjiang** [epam/ketcher#1248] [Q&A] [answered]
mode standalone js file too large，main.js 7mb, 
Causes the ketcher to load too slowly
Is there any way to reduce or reduce the size of js file by reducing related components
thanks
<img width="1014" alt="WechatIMG726" src="https://user-images.githubusercontent.com/42111602/153405797-6fe5e4be-577a-473a-adfc-64c04e627b0b.png">

**Comment by AndreiMazol** [comment, marked as answer]
Dear @niuzhenjiang ,

standalone version is based on [indigo-ketcher](https://www.npmjs.com/package/indigo-ketcher) package that  is generated from Indigo code base. This package contains only required functions so it is impossible to reduce its size. 

Regards,
Andrei

**Reply by niuzhenjiang** [reply]
I got it, thanks a lot
